core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -560,16 +561,15 @@ class LoggingMessageFormatter extends MessageFormatter with LazyLogging {
 }
 
 class NoOpMessageFormatter extends MessageFormatter {
-  override def init(props: Properties): Unit = {}
 
   def writeTo(consumerRecord: ConsumerRecord[Array[Byte], Array[Byte]], output: PrintStream): Unit = {}
 }
 
 class ChecksumMessageFormatter extends MessageFormatter {
   private var topicStr: String = _
 
-  override def init(props: Properties): Unit = {
-    topicStr = props.getProperty("topic")
+  override def configure(configs: Map[String, _]): Unit = {
+    topicStr = configs.get("topic").toString
```

Comment:
do we risk NPE here?

Reply:
can we add a test?

Reply:
Good catch! I've pushed a change to address it